README.md

```diff
@@ -201,6 +204,49 @@ already running.
     $ pgenv restart
     PostgreSQL restarted
 
+### pgenv available
+
+Shows all the versions of PostgreSQL available to download and build. Handy to
+find a version you want to pass to the `build` command. Note that this command
+produces copious output.
+
+    $ pgenv available
+    ...
+            Available PostgreSQL versions
+    ==============================================
+
+                    PostgreSQL 9.6
+    -----------------------------------------------
+    9.6.0	 9.6.1	 9.6.2	 9.6.3	 9.6.4	 9.6.5
+    9.6.6	 9.6.7	 9.6.8	 9.6.9	 9.6.10
+
+                    PostgreSQL 10
+    -----------------------------------------------
+    10.0	 10.1	 10.2	 10.3	 10.4	 10.5
+
+                    PostgreSQL 11
+    -----------------------------------------------
+    11beta1	 11beta2	 11beta3
+
+The versions are organized by major release version. Any listed version may be
+passed to `pgenv` commands that require a version.
+
+To limit the list to specific major releases, pass those releases to
+`available`. For example, to list only the `9.6` and `10` available versions:
+
+    $ pgenv available 9.6 10
+            Available PostgreSQL versions
+    ==============================================
+
+                    PostgreSQL 9.6
+    -----------------------------------------------
+    9.6.0	 9.6.1	 9.6.2	 9.6.3	 9.6.4	 9.6.5
```

Comment:
Hrm. I wonder if we'd be better off just putting three spaces after each number, rather than a tab — since tab length can vary.

Reply:
That sounds fine, but we need to keep the string printed in fixed length, so use printf. If you cherry pick commit ca8ff60c636514cd5ea45ec6097120b382b632ee I suspect this can be merged.
Please note I left tabs in the check command output.